### DISCUSSION TITLE
Lore Support and Smooth Custom Items Search

### DISCUSSION THREAD

**Discussion by slipstricker** [Zrips/CMI#3490] [Feature Requests]
Create a comand named /recipe gui so players can see all crafts that the server created (not vanila ones), the player will have a permission cmi.command.recipegui and will be able to see all custom crafts. Also add support to lore items
---
This is necessary bcs it will help the player that wants to see the custom crafts without knowing the name and will see only the custom ones, not vanila, also lore will give more options to edit items. I believe it will be a nice feature if added.

**Comment by CloudeLecaw** [comment]
We allready have a custom recripe editor /cmi customrecipe.
This editor may could need a permission like for a overview like
**_cmi.command.customrecipe.list_**

With this permission a chatgui should open where players can see all recipes
The customrecipe editor may could get a new argument **_/cmi customrecipe list_**
If entering then players can see the gui with all custom recipes but they cant edit/add or remove anything

It would be the owners job now simply a customalias that is executing /cmi customrecipe list like /recipegui ... or what ever you wanted


I see no reason for creating a whole new command to this. This also would fit, because cmi recipe allready exists (it shows recipe of a single crafting if you know the name of the item like /cmi recipe rails - It needs item idname not the showed name based in your client.) 
maybe if you created a customrecipe with the name "Testitem" then it would be nice if recipe may could show the single recipe by typing /cmi recipe Testitem to show hwo to craft it

What you think about this idea?

**Comment by slipstricker** [comment]
the /cmi customrecipe list
command does not work

**Comment by myavuzokumus** [comment]
This feature should definitely be.

**Reply by VIQingNiao** [reply]
the /cmi customrecipe list
command does not work

**Comment by CloudeLecaw** [comment]
last Update a Argumente to tbe recipe command was added to show cmis custom recipes
